tools/check_format.py

```diff
@@ -177,8 +177,8 @@ def checkFormatVisitor(arg, dir_name, names):
   if os.path.isfile(target_path):
     checkFormat("./" + target_path)
   else:
-    os.chdir(target_path)
-    os.path.walk(".", checkFormatVisitor, None)
+    # os.chdir(target_path)
```

Comment:
please remove commented out code

Reply:
Hello Matt , thanks for looking at it . pushed a fix with removing commented code . Yes , it fixes #2391 , will add that to description . can you please elaborate on what fixing DCO means ?